pkg/phlaredb/symdb/stacktrace_tree_test.go

```diff
@@ -220,9 +222,48 @@ func Benchmark_stacktrace_tree_insert(b *testing.B) {
 	b.ReportAllocs()
 
 	for i := 0; i < b.N; i++ {
-		x := newStacktraceTree(0)
+		x := newStacktraceTree(defaultStacktraceTreeSize)
 		for j := range p.Sample {
 			x.insert(p.Sample[j].LocationId)
 		}
 	}
 }
+
+func Benchmark_stacktrace_tree_insert_default_sizes(b *testing.B) {
+	if os.Getenv("COMPARE_STACKTRACETREE_INSERT_DEFAULT_SIZES") != "true" {
+		b.Skip("This benchmark is only used for comparing different initial tree sizes")
+	}
```

Comment:
Let's remove the env var check

Reply:
And always skip the benchmark, or remove the call to `b.Skip` as well?

Reply:
`b.Skip` as well – let's run the benchmark as usual